aspnetcore/client-side/libman/libman-cli.md

```diff
@@ -0,0 +1,498 @@
+libman restore [-h|--help]
+```
+
+### Options
+
+The following options can be appended to the `libman restore` command:
+
+[!INCLUDE [standard-cli-options](../../includes/libman-cli/standard-cli-options.md)]
+
+### Examples
+
+To restore the library files defined in *libman.json*:
+
+```console
+libman restore
+```
+
+## Delete library files
+
+The `libman clean` command deletes library files previously restored via LibMan. Folders that become empty after this operation are deleted. The library files' associated configurations in the `libraries` property of *libman.json* aren't removed.
+
+### Synopsis
+
+```console
+libman clean [--verbosity]
+libman clean [-h|--help]
+```
+
+### Options
+
+The following options can be appended to the `libman clean` command:
+
+[!INCLUDE [standard-cli-options](../../includes/libman-cli/standard-cli-options.md)]
+
+### Examples
+
+To delete library files installed via LibMan:
+
+```console
+libman clean
+```
+
+## Uninstall library files
+
+The `libman uninstall` command:
+
+* Deletes all files associated with the specified library from the destination in *libman.json*.
+* Removes the associated library configuration from *libman.json*.
+
+An error occurs when:
+
+* No *libman.json* file exists in the project root.
+* The specified library doesn't exist.
+
+If more than one library with the same name is installed, you're prompted to choose one.
+
+### Synopsis
+
+```console
+libman uninstall <LIBRARY> [--verbosity]
+libman uninstall [-h|--help]
+```
+
+### Arguments
+
+`LIBRARY`
+
+The name of the library to uninstall. This name may include version number notation (for example, `@1.2.0`).
+
+### Options
+
+The following options can be appended to the `libman uninstall` command:
+
+[!INCLUDE [standard-cli-options](../../includes/libman-cli/standard-cli-options.md)]
+
+### Examples
+
+Consider the following *libman.json* file:
+
+[!code-json[](samples/LibManSample/libman.json)]
+
+* To uninstall jQuery, either of the following commands succeed:
+
+  ```console
+  libman uninstall jquery
+  ```
+
+  ```console
+  libman uninstall jquery@3.3.1
+  ```
+
+* To uninstall the Lodash files installed via the `filesystem` provider:
+
+  ```console
+  libman uninstall C:\temp\lodash\
+  ```
+
+## Update library version
+
+The `libman update` command updates a library installed via LibMan to the specified version.
+
+An error occurs when:
+
+* No *libman.json* file exists in the project root.
+* The specified library doesn't exist.
+
+If more than one library with the same name is installed, you're prompted to choose one.
+
+### Synopsis
+
+```console
+libman update <LIBRARY> [-pre] [--to] [--verbosity]
+libman update [-h|--help]
+```
+
+### Arguments
+
+`LIBRARY`
+
+The name of the library to update.
+
+### Options
+
+The following options can be appended to the `libman update` command:
+
+* `-pre`
+
+  Flag whose presence indicates the latest pre-release version of the library will be downloaded.
```

Comment:
Dash? "prerelease" ... given Chicago.

Future tense and the sentence seems awkward ("whose presence") ...

> Obtain the latest prerelease version of the library.